app/views/amusement_parks/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1><%=@amusement_park.name%> Amusement Park</h1>
+<p>Admission Cost: <%= @amusement_park.admission_cost%></p>
+
+<div id="mechanics_working">
+  <h2>Mechanics Working Here:</h2>
+  <% @amusement_park.mechanics.distinct.each do |mechanic| %>
```

Comment:
We don't want to do logic in our view. Instead we would want to create a method in our `amusement_park` model where we can call `mechanics.distinct`. Then we would just call that method in here on our `@amusement_park` instance variable.